test/TestUtilities/GraphicsUtil.cpp

```diff
@@ -16,4 +17,14 @@ namespace sf
 
         return stream.str().c_str();
     }
+
+    doctest::String toString(const sf::Transform& transform)
+    {
+        std::ostringstream stream;
+        stream << transform.getMatrix()[0] << ", " << transform.getMatrix()[4] << ", " << transform.getMatrix()[12] << ", ";
+        stream << transform.getMatrix()[1] << ", " << transform.getMatrix()[5] << ", " << transform.getMatrix()[13] << ", ";
+        stream << transform.getMatrix()[3] << ", " << transform.getMatrix()[7] << ", " << transform.getMatrix()[15];
```

Comment:
Maybe store the matrix in a variable for readability?

Reply:
Good idea. Fixed.